test/serializable_test.dart

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@TestOn('vm || browser')
```

Comment:
when running these tests on the VM and the browser .. with the added reflectable transformer I get
```
 00:03 +199 -1: [VM] loading test/serializable_test.dart [E]
      Failed to load "test/serializable_test.dart": Unsupported operation: The transformed code is running with the untransformed reflectable package. Remember to set your package-root to 'build/.../packages'.
      http://localhost:62337/serializable_test.dart 28:5               _initializeReflectable
      http://localhost:62337/serializable_test.dart 18:3               main
      ===== asynchronous gap ===========================
      package:test                                                     internalBootstrapVmTest
      http://localhost:62337/serializable_test.dart.vm_test.dart 8:13  main
```
I couldn't figure out how to set the package-root, so just avoided the problem by not running the test on VM.